packages/runtime/src/templates/handlerUtils.ts

```diff
@@ -107,24 +105,26 @@ export const augmentFsModule = ({
   const statsOrig = promises.stat
   // ...then money-patch it to see if it's requesting a CDN file
   promises.readFile = (async (file, options) => {
-    const base = getBase()
+    // In production use the public URL (e.g. https://example.com). Otherwise use the deploy URL, e.g. https://deploy-preview-123--example.netlify.app
+    const baseUrl = process.env.CONTEXT === 'production' ? process.env.URL : process.env.DEPLOY_PRIME_URL
+
     // We only care about page files
     if (file.startsWith(pageRoot)) {
-      // We only want the part after `pages/`
+      // We only want the part after `.next/server/`
```

Comment:
again, this is because it could be coming from `pages` or the `app` dir correct?

Reply:
Exactly